adafruit_platformdetect/board.py

```diff
@@ -21,14 +21,18 @@
 """
 
 # imports
-from __future__ import annotations
 import os
 import re
-from typing import Optional, TYPE_CHECKING
+
+try:
+    from typing import TYPE_CHECKING, Optional
+except ImportError:
+    TYPE_CHECKING = False
+
 from .constants import boards, chips
 
 if TYPE_CHECKING:
-    from . import Detector
+    from .protocols import DetectorProtocol
 
 __version__ = "0.0.0-auto.0"
 __repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_PlatformDetect.git"
```

Comment:
@dhalbert or @makermelissa can you give me some context as to why these lines are included in `__init__.py`, `board.py`, and `chip.py` (vs. just having it in `__init__.py`)?  Just wondering if these should be added to any new files I add (like `protocols.py` that I added in this draft)